hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -6,13 +6,13 @@
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
   entries: [
     {
       name: smoke
       desc: '''
             Basic smoke test accessing a major datapath within the keymgr.
-            Most of other sequences are derived from this to have similar init and sequence.
```

Comment:
For line 14, the new template removed he "Basic" wording, I think we can just use "Smoke test".

Reply:
Updated, thanks